docs/content/download/default.yml

```diff
@@ -32,6 +32,13 @@ body:
        * jq is in the official [Arch](https://archlinux.org/packages/?q=jq) repository.
          Install using `sudo pacman -S jq`.
 
+       * jq 1.7 binaries for
+         [AMD 64-bit](https://github.com/jqlang/jq/releases/download/jq-1.7/jq-linux-amd64)
+         or
+         [ARM 64-bit](https://github.com/jqlang/jq/releases/download/jq-1.7/jq-linux-arm64)
+         or
+         [AMD 32-bit](https://github.com/jqlang/jq/releases/download/jq-1.7/jq-linux-i386).
```

Comment:
Maybe call this i386 now?

Reply:
I wonder if we should update the old release link texts from "64-bit" to "AMD 64-bit" etc to be clearer?

Reply:
I standardized the arch types to AMD64, ARM64 & i386 to match our built artifacts in the release to avoid confusion: https://github.com/jqlang/jq/pull/2879/commits/1098c801110f25ddf9862fa268005308a9daf7af